src/test/java/com/microsoft/sqlserver/jdbc/unit/statement/StatementTest.java

```diff
@@ -1111,20 +1111,20 @@ public void testJdbc41CallableStatementMethods() throws Exception {
                                 + "(?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)}")) {
                     cstmt.registerOutParameter(1, java.sql.Types.VARCHAR);
                     cstmt.registerOutParameter(2, java.sql.Types.INTEGER);
-                    cstmt.registerOutParameter(3, java.sql.Types.FLOAT);
-                    cstmt.registerOutParameter(4, java.sql.Types.DECIMAL);
+                    cstmt.registerOutParameter("col3Value", java.sql.Types.FLOAT);
```

Comment:
Why are these changes necessary?

Reply:
code coverage :)
this always calls the same registerOutParameter it's good to test the others as well